src/main/java/com/solace/samples/features/distributedtracing/manualinstrumentation/TopicPublisher.java

```diff
@@ -51,11 +54,12 @@
  */
 public class TopicPublisher {
 
-  final String TOPIC_NAME = "solace/samples/jms/direct/pub/tracing";
-  private static final String SERVICE_NAME = "SolaceJMSTopicPublisherManualInstrument";
+  private static final String SERVICE_NAME = "ACME Product Master [DEV]";
```

Comment:
Samples deleted and `build.gradle` updated for the generated start scripts.

Reply:
@itsJamilAhmed @TamimiGitHub `TopicSubscriber.java` and `TopicSyncReceiver.java` might not create a span from the broker (no acknowledgement w/ Direct), but they do show how to create a child span in the client side app. I do feel like that provides value...just b/c the broker doesn't create a span doesn't mean the app itself shouldn't: https://github.com/SolaceSamples/solace-samples-jms/pull/58/files#diff-4df0a9e60ece245ad8b3bf5efb3c0fcc36b751bda574db637f4c059a21a7aba6L134

Reply:
The `QueueSubscriber.java` example covers off this case no? In fact it goes one better to create a `receiveSpan` and then a `processingSpan` as another child.

Reply:
@itsJamilAhmed You raise a very good point...I agree that if it's the same thing over and over again then it's not useful. Maybe we should just rename the sample to `SubscriberWithTracing.java` and then wouldn't the `QueuePublisher` and `TopicPublisher` also be redundant? Should we just have one that is `PublisherWithTracing` ?

Reply:
I do agree with renaming the file names for clarity and not have the same file names as found in the patterns dir. 

And @itsJamilAhmed the `build.gradle` file was not committed

Reply:
I also realized we will need to remove `[HowToImplementTracingManualInstrumentation.java](https://github.com/SolaceSamples/solace-samples-jms/blob/7ac986d11283b9cf73e39bd7c1b254fea7b9fde0/src/main/java/com/solace/samples/jms/snippets/HowToImplementTracingManualInstrumentation.java)`

Reply:
I agree with doing a rename on the samples to make the DT ones more specific. (The JCSMP samples are clearer in this regard btw.)
Also on having both a `TopicPublisher` and `QueueSubscriber`, there is some redundancy yes when the focus here is to show the instrumentation, not how to publish a message to either type... 

An argument for keeping it would be around convenience of running the samples. Publishing to a topic needs the user to have configured subscriptions on the queue in order for the queue subscriber to work. Publishing direct to the queue and subsequently subscribing from it makes things more 'out of the box' ready. Conversely, publishing to a topic is the best practice so that should take priority?

Reply:
> I also realized we will need to remove [HowToImplementTracingManualInstrumentation.java](https://github.com/SolaceSamples/solace-samples-jms/blob/7ac986d11283b9cf73e39bd7c1b254fea7b9fde0/src/main/java/com/solace/samples/jms/snippets/HowToImplementTracingManualInstrumentation.java)

I don't think we should remove that one. I actually prefer that which gets straight to the point vs. these longer apps where people have to look through more code ot understand what is going on.

Reply:
Fair enough yeah that could be used for snippet references